RomanToArabicNumbersChanger.java

```diff
@@ -0,0 +1,69 @@
+public class RomanToArabicNumbersChanger {
+
+    private int changeRomanToArabicNumber(String romanNumber) throws RomanException {
+        int current = 0;
+        int previous = 0;
+        int sum = 0;
+        romanNumber = romanNumber.toUpperCase();
+
+        if (romanNumber.matches("[0-9]") || !romanNumber.matches("[^MDCLXVI]") || !romanNumber.matches("[MDCLXVI]{0,3}")) {
+            throw new RomanException();
+        } //nie dziala to jeszcze:(
```

Comment:
If something is not working make comment //TODO -> Intellij will highlight it. What is more insert some more informations like what is suppose to do. If another software engineer will take this code he is not going to know how it suppose to work